frontend/src/components/MeetingRoom/ReportIssueButton/ReportIssueButton.tsx

```diff
@@ -3,42 +3,47 @@ import { ReactElement, useRef } from 'react';
 import FeedbackIcon from '@mui/icons-material/Feedback';
 import { blue } from '@mui/material/colors';
 import ToolbarButton from '../ToolbarButton';
-import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
 export type ReportIssueButtonProps = {
   handleClick: () => void;
   isOpen: boolean;
+  isOverflowButton?: boolean;
 };
 
 /**
  * ReportIssueButton Component
  *
  * Displays a clickable button to open/close the ReportIssue panel.
  * @param {ReportIssueButtonProps} props - The props for the component.
+ *  @property {Function} handleClick - click handler to open the Report Issue panel
```

Comment:
just curious why do we do `{(event?: MouseEvent | TouchEvent) => void}` passed in as a parameter for `ArchivingButton` component but not for this one or some other buttons?